client/service/test/src/client/mod.rs

```diff
@@ -49,9 +49,10 @@ use sp_consensus::{
 	BlockOrigin, SelectChain, BlockImport, Error as ConsensusError, BlockCheckParams, ImportResult,
 	BlockStatus, BlockImportParams, ForkChoiceStrategy,
 };
+use sp_finality_grandpa::GRANDPA_ENGINE_ID;
```

Comment:
We could create a `TEST_ENGINE_ID` here instead of adding a dependency on `sp_finality_grandpa`.